src/main/java/org/terasology/joshariasSurvival/systems/WorkstationDiagnosticsSystem.java

```diff
@@ -161,7 +149,7 @@ public String logItemCraftingComplexity() {
         }
 
         for (Map.Entry<ResourceUrn, Integer> entry : maxDepthMap.entrySet()) {
-            logger.info(entry.getKey().toString() + " MaxDepth=" + entry.getValue() + " MaxComplexity=" + maxComplexityMap.get(entry.getKey()));
+            logger.info("{} MaxDepth={} MaxComplexity={}", entry.getKey().toString(), entry.getValue(), maxComplexityMap.get(entry.getKey()));
```

Comment:
I think we can omit the `toString()` call here
```suggestion
            logger.info("{} MaxDepth={} MaxComplexity={}", entry.getKey(), entry.getValue(), maxComplexityMap.get(entry.getKey()));
```